source/faucet/main.d

```diff
@@ -374,9 +374,10 @@ public class Faucet : FaucetAPI
     }
 
     /// POST: /send_transaction
-    public override void sendTransaction (string recv, ulong amount)
+    public override void sendTransaction (string recv)
     {
         PublicKey pubkey = PublicKey.fromString(recv);
+        ulong amount = 100;
         Amount leftover = amount.coins;
```

Comment:
Aren't `leftover` and `amount` redundant ?

Reply:
Yes, and I also thought of removing it.
I kept it for the logging in L397 and L427, in case we change the value. The `leftover` would be 100 * the smaller units.